driver.sh

```diff
@@ -139,6 +139,13 @@ setup_variables() {
       export ARCH=powerpc
       export CROSS_COMPILE=powerpc64le-linux-gnu- ;;
 
+    "s390")
+        config=defconfig
+        image_name=bzImage
+        using_qemu=false
```

Comment:
right but i meant removing this

Reply:
If we remove that from `driver.sh`, it requires a local user to explicitly add the `BOOT=0`, otherwise the build won’t even start because QEMU won’t be found.